<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataInputDialog</class>
 <widget class="QDialog" name="DataInputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>239</width>
    <height>195</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ok_pb">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="q1_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Q1:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="q1_le">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="q2_le">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="q2_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Q2:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="bit_r_le">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bit_r_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bit R:</string>
   </property>
  </widget>
  <widget class="QLabel" name="file_path_lbl">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Monospace</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Path:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="file_path_le">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="file_browser_pb">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
